src-docs/src/views/popover/popover_example.js

```diff
@@ -47,6 +47,61 @@ import PopoverFixed from './popover_fixed';
 const popoverFixedSource = require('!!raw-loader!./popover_fixed');
 const popoverFixedHtml = renderToHtml(PopoverFixed);
 
+const popOverSnippet = `<EuiPopover
+  button={button}
+  closePopover={this.closePopover.bind(this)}>
```

Comment:
Let's remove the `.bind(this)` most consumers should use the arrow method and it will clean up the snippet and make it consistent with our other snippets.
```suggestion
  closePopover={this.closePopover}>
```

Reply:
I think we also need to be adding `isOpen={this.state.isPopoverOpen}` to all of these or else they may not know how to get the thing to actually show.